client/dashboard/sites/deployments-list/dataviews/views.ts

```diff
@@ -1,21 +1,20 @@
-import type { SortDirection, View } from '@wordpress/dataviews';
+import type { SortDirection, SupportedLayouts, View } from '@wordpress/dataviews';
 
 export const DEFAULT_VIEW: View = {
 	type: 'table',
-	perPage: 25,
```

Comment:
This is not one of the default values:

<img width="345" height="92" alt="Image" src="https://github.com/user-attachments/assets/d61a2c43-81a9-458d-95d1-66faea5ec996" />